comit/src/orderpool.rs

```diff
@@ -22,6 +23,22 @@ pub struct OrderPool {
     ///
     /// Allows us to filter out our own orders.
     me: PeerId,
+
+    /// A cache for storing which orders don't match.
```

Comment:
Can you explain more the functionality you are trying to achieve?

If an order does not match at a given point in time, it may match if the user adds a new order in their orderbook.

Reply:
Which is why the key that is stored is a tuple of the two orders that don't match.

The idea was, if two orders don't match, I can store that for later and avoid checking them again in the next round. The primary motivation was reducing the log-spam of the same two orders constantly not matching over and over again.

Reply:
thanks!